ElfUtils/ElfFile.cpp

```diff
@@ -200,10 +217,25 @@ std::string ElfFileImpl<ElfT>::GetFilePath() const {
   return file_path_;
 }
 
+template <typename ElfT>
+ErrorMessageOr<LineInfo> ElfUtils::ElfFileImpl<ElfT>::GetLineInfo(uint64_t address) {
```

Comment:
If I got it right, this should only be called, if has_debug_info_section_ is present. So maybe add a CHECK here.

Reply:
done.